### DISCUSSION TITLE
srsUE is running on ZCU102 or ZCU111

### DISCUSSION THREAD

**Discussion by everytt** [srsran/zynq_timestamping#61] [General]
https://srsran.github.io/zynq_timestamping/index.html
The figure here shows that srsUE can run on a ZCU111 or ZCU102.
Is this possible?

![image](https://github.com/srsran/zynq_timestamping/assets/33945009/3b3895b9-9fed-4672-be58-4934e0ce4228)

**Comment by ofontbach** [comment]
Hi @everytt,

Yes. That's correct. Baseline projects to do that can be found in the repo:

- zcu102: https://github.com/srsran/zynq_timestamping/tree/main/projects/zcu102
- zcu111: https://github.com/srsran/zynq_timestamping/tree/main/projects/zcu111

Those are thought to run the UE in a fully embedded way, for which the performance will be quite limited without further accelerating the PHY in the FPGA.

Cheers
